packages/teleterm/src/mainProcess/windowsManager.ts

```diff
@@ -0,0 +1,96 @@
+import { BrowserWindow, Rectangle, screen } from 'electron';
+import { getAssetPath } from 'teleterm/mainProcess/runtimeSettings';
+import path from 'path';
+import { FileStorage } from 'teleterm/services/fileStorage';
+import { RuntimeSettings } from 'teleterm/mainProcess/types';
+import theme from 'teleterm/ui/ThemeProvider/theme';
+
+type WindowState = Rectangle;
```

Comment:
I wanted to store maximised/fullscreen info as well, but I couldn't force it to work reliably when using multiple displays.
When I close the app on the second screen I would to see it on the same screen when it is opened. Unfortunately, it opens on the first screen when the app was closed in maximised/full screen state.

But THB, I'm not sure it's that important. Apple native apps/Chrome don't work that way. When you close a maximized app, it will open at its normal size.